samples/config_helper_dawn.cc

```diff
@@ -23,8 +23,7 @@ ConfigHelperDawn::ConfigHelperDawn() = default;
 
 ConfigHelperDawn::~ConfigHelperDawn() = default;
 
-namespace {
-// Callback which prints a message from a Dawn device operation.
+namespace {  // Callback which prints a message from a Dawn device operation.
```

Comment:
Move the comment down to above the method. I thought it was a namespace comment initially.